content/community/contributing-code/Test Pull Requests.adoc

```diff
@@ -0,0 +1,58 @@
+---
+Title: Testing Pull Requests
+Weight: 40
+---
+
+:imagesdir: ./../../../../images/en/community
+
+== Testing Pull Requests
```

Comment:
Remove this subtitle because the same phrase appears as a level 1 title (because it is in the front-matter).